src/main.rs

```diff
@@ -10,5 +12,13 @@ async fn main() {
         .with_max_level(tracing::Level::INFO)
         .with_ansi(true)
         .init();
-    bootstrap_application().await;
+    let app_handle = tokio::spawn(async move {
+        build_websocket_client().await;
+    });
+    let http_handle = tokio::spawn(async move {
+        build_healthcheck_server()
+            .await
+            .expect("Failed to start HTTP server");
+    });
+    futures::future::join_all(vec![app_handle, http_handle]).await;
```

Comment:
also i think its very possible for the app to crash but this will wait forever for the http server to stop, so perhaps not the best health check

Reply:
Yeah it's not really a good healthcheck, but Lightsail just needs a http endpoint on the service